lxd/network_acls.go

```diff
@@ -155,9 +156,18 @@ func networkACLsGet(d *Daemon, r *http.Request) response.Response {
 		return response.InternalError(err)
 	}
 
+	userHasReBACPermission, err := getReBACPermissionChecker(s, r, openfga.Viewer, openfga.NetworkACL)
+	if err != nil {
+		return response.SmartError(err)
+	}
+
 	resultString := []string{}
 	resultMap := []api.NetworkACL{}
 	for _, aclName := range aclNames {
+		if !userHasReBACPermission(fmt.Sprintf("%s_%s", projectName, aclName)) {
```

Comment:
There were some helper functions in a previous commit that performed the string concatenation, should we have another one for ACLs?